packages/muelu/src/Transfers/BlockedTransfers/MueLu_ReorderBlockAFactory_decl.hpp

```diff
@@ -84,10 +84,8 @@ namespace MueLu {
     //@{
 
     //! Constructor.
-    ReorderBlockAFactory() { }
+    ReorderBlockAFactory() = default;
```

Comment:
You can even delete this one as well. Then, the compiler will generate the default constructor by itself.